sysconfigs/daint/compilers.yaml

```diff
@@ -117,19 +102,6 @@ compilers:
     modules: []
     environment: {}
     extra_rpaths: []
-- compiler:
-    spec: gcc@8.1.0
-    paths:
-      cc: /opt/gcc/8.1.0/bin/gcc
```

Comment:
the implememtatiom using the path instead of the PrgEnv was recommended by CSCS after we discussed with them, if I understand what you are doing here, is to change compilers to being package, is it correct ?

Reply:
Didn't know that. I was under impression that modules were the preferred way. I'll change it back then.

Reply:
No, that one was to remove redundant entry (there were 2 entries for the same compiler).

Reply:
2 entries for the same compilers, is indeed a mistake